Clean/Table/Basic.lean

```diff
@@ -156,6 +164,13 @@ instance : Repr (Cell W S) where
     | .input off => ".input " ++ reprStr off
     | .aux i => ".aux " ++ reprStr i
 
+instance : ToJson (Cell W S) where
+  toJson cell := match cell with
+    | .input off => toJson off
```

Comment:
nit: the idiomatic way to define functions whose arguments are immediately matched on is this:
```suggestion
  toJson
    | .input off => toJson off
```